### PR TITLE
[PLT-6185] Typography Update

### PR DESCRIPTION
**Pull request by jordanell**
PLT-6185

## Description

1. Updated typography to spec
2. Updated docs

## Notes

Key highlight ares:

1. Typography variant "caption" removed
2. Typography variant "subtitle1" removed
3 Typography variant "subtitle2" removed
4. Typography variant "button" removed
5. Added `fontWeight` as a prop for Typography

## Where to Start

- `<Typography>` component

### CODE REVIEW

File: packages/riipen-ui-docs/src/pages/components/typography/Color.jsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import React from "react";
+
+import Typography from "riipen-ui/components/Typography";
+
+export default function Color() {
+  return (
+    <div>
+      <Typography color="initial" gutter>
+        Initial
+      </Typography>
```
Comment (bolinkd) [inline packages/riipen-ui-docs/src/pages/components/typography/Color.jsx:10]: are we moving to one singular color for text? I know we use the _muted_ variant a fair bit currently and there is no real replacement in these new colors

Comment (jordanell) [reply packages/riipen-ui-docs/src/pages/components/typography/Color.jsx:10]: That's a good question. Maribel didn't provide colors in her latest guide. I am going to check with her before we merge this.

Comment (jordanell) [reply packages/riipen-ui-docs/src/pages/components/typography/Color.jsx:10]: @bolinkd check out https://github.com/riipen/ui/pull/92/commits/3cb51ea5c95d334d56da01af1862d25a0513168d !

File: packages/riipen-ui-docs/package.json [hunk 2]
```diff
@@ -57,6 +57,7 @@
     "@babel/core": "^7.10.5",
     "@babel/node": "^7.6.3",
     "audit-ci": "3.0.1",
+    "chokidar": "^3.4.3",
```
Comment (bolinkd) [inline packages/riipen-ui-docs/package.json:60]: I dont see this used anywhere, was it intentional?

Comment (jordanell) [reply packages/riipen-ui-docs/package.json:60]: Ya intentional. Apparently nextjs is having issues with ubuntu in their latest patch and manually forcing this install in dev-dependencies fixes it.

**Review by bolinkd — APPROVED** [review]
Looks good overall, there will be a substantial number of changes needed on the web to make everything work with these changes 🙈

**Pull request merged**